azurite-containers.js

```diff
@@ -0,0 +1,49 @@
+const {
```

Comment:
Can we add a comment with a little description explaining why this file is needed and what problem does it solves?